coordinator/src/test/scala/filodb.coordinator/queryplanner/MultiPartitionPlannerSpec.scala

```diff
@@ -923,94 +923,82 @@ class MultiPartitionPlannerSpec extends AnyFunSpec with Matchers with PlanValida
-          PartitionAssignment("remote1", "remote-url", TimeRange(startSeconds * 1000 - lookbackMs,
-            localPartitionStartMs - 1)), PartitionAssignment("remote2", "remote-url",
-            TimeRange(localPartitionStartMs, endSeconds * 1000)))
-        else Nil
-      }
-
-      override def getMetadataPartitions(nonMetricShardKeyFilters: Seq[ColumnFilter], timeRange: TimeRange): List[PartitionAssignment] = List(
-        PartitionAssignment("remote1", "remote-url", TimeRange(startSeconds * 1000 - lookbackMs,
-          localPartitionStartMs - 1)), PartitionAssignment("remote2", "remote-url",
-          TimeRange(localPartitionStartMs, endSeconds * 1000)))
-
-    }
-    val engine = new MultiPartitionPlanner(partitionLocationProvider, localPlanner, "local", dataset, queryConfig)
-    val lp = Parser.queryRangeToLogicalPlan("test{job = \"app\"}", TimeStepParams(startSeconds, step, endSeconds))
-
-    val promQlQueryParams = PromQlQueryParams("test{job = \"app\"}", startSeconds, step, endSeconds)
-
-    val execPlan = engine.materialize(lp, QueryContext(origQueryParams = promQlQueryParams, plannerParams =
-      PlannerParams(processMultiPartition = true)))
-    val stitchRvsExec = execPlan.asInstanceOf[StitchRvsExec]
-    stitchRvsExec.children.size shouldEqual (2)
-    stitchRvsExec.children(0).isInstanceOf[PromQlRemoteExec] shouldEqual (true)
-    stitchRvsExec.children(1).isInstanceOf[PromQlRemoteExec] shouldEqual (true)
-
+    execPlan.isInstanceOf[PromQlRemoteExec] shouldEqual (true)
 
-    val remoteExec = stitchRvsExec.children(0).asInstanceOf[PromQlRemoteExec]
+    // Instant/Raw queries will have same start and end point
+    val remoteExec = execPlan.asInstanceOf[PromQlRemoteExec]
     val queryParams = remoteExec.queryContext.origQueryParams.asInstanceOf[PromQlQueryParams]
     queryParams.startSecs shouldEqual startSeconds
-    queryParams.endSecs shouldEqual (localPartitionStartMs - 1) / 1000
+    queryParams.endSecs shouldEqual endSeconds
     queryParams.stepSecs shouldEqual step
     remoteExec.queryContext.plannerParams.processFailure shouldEqual true
     remoteExec.queryContext.plannerParams.processMultiPartition shouldEqual false
-    remoteExec.queryEndpoint shouldEqual "remote-url"
-
-    val remoteExec2 = stitchRvsExec.children(1).asInstanceOf[PromQlRemoteExec]
-    val queryParams2 = remoteExec2.queryContext.origQueryParams.asInstanceOf[PromQlQueryParams]
-    queryParams2.startSecs shouldEqual endSeconds
-    queryParams2.endSecs shouldEqual endSeconds
-    queryParams2.stepSecs shouldEqual step
-    remoteExec2.queryContext.plannerParams.processFailure shouldEqual true
-    remoteExec2.queryContext.plannerParams.processMultiPartition shouldEqual false
-    remoteExec2.queryEndpoint shouldEqual "remote-url"
-
+    remoteExec.queryEndpoint shouldEqual "remote-url2"
   }
 
+//  // TODO: what is this the purpose of this test? The lookback is 300s, and the assignment
```

Comment:
Why comment out the test?

Reply:
I was hoping others would know what this test was for-- it looks like we used to expect partition-split plans would always return at least one data-point per partition, regardless of lookback/offset. That doesn't jive with the way we handle partition-splits elsewhere in the planners, so I think this test should be removed.

The reason it passed prior to this PR is because the MPP sent `PeriodicSeries` and `RawSeries` through the old `materializePeriodicAndRawSeries` logic. That always materialized at least one plan per partition.

`materializePeriodicAndRawSeries` was built to materialize plans with equality-only filters, so that would not work for this PR. Instead, plans are:
- if split, passed to the regex-handling `materializeSplitLeafPlan`, which accounts correctly for lookback.
- if not split, forwarded as-is to any partitions that own the query data.